3581/3/app/controllers/places_controller.rb

```diff
@@ -0,0 +1,14 @@
+class PlaceController < ApplicationController
+  get '/' do
+    @places = Place.all
+    erb :index
+  end
+
+  get '/places/:id' do
+    @place = Place.find(params[:id])
+    @reviews = @place.reviews.reverse
+    @average_score = @place.reviews.average(:grade).to_f.truncate(1)
```

Comment:
do you really need cast to float?
did you test this?
https://apidock.com/rails/String/truncate
string method

Reply:
Yep, because when i call AVG on grade, it returns BigDecimal. Moreover i have to show this value in format 1.2(e.g.), so i need to convert it to float and truncate unnecessary digits after point.
Perhaps, you could propose another way to calculate Average Rating?